packages/drawer/rc/__stories__/index.story.js

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import Button from '@pluralsight/ps-design-system-button'
+import * as Icon from '@pluralsight/ps-design-system-icon'
+import * as Text from '@pluralsight/ps-design-system-text'
+import Row from '@pluralsight/ps-design-system-row'
+
+import { Drawer } from '../index.js'
+
+const DrawerHeadContent = props => (
+  <Text.P {...props} style={{ padding: '10px 0' }} />
+)
+
+const DrawerBodyContent = props => <Text.P {...props} style={{ padding: 20 }} />
+
+const ControlledExternalState = () => {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <>
+      <button
+        onClick={() => {
+          setOpen(true)
+        }}
+      >
+        open
+      </button>
+      <button
+        onClick={() => {
+          setOpen(false)
+        }}
+      >
+        close
+      </button>
+      <Drawer
+        isOpen={open}
+        onToggle={() => {
+          setOpen(!open)
+        }}
+      >
+        <Drawer.Head>
+          <DrawerHeadContent>Click me to open</DrawerHeadContent>
+        </Drawer.Head>
+        <Drawer.Body>
+          <DrawerBodyContent>Drawer Content here</DrawerBodyContent>
+        </Drawer.Body>
+      </Drawer>
+    </>
+  )
+}
+
+const ControlledStartOpen = () => {
+  const [open, setOpen] = useState(true)
+  return (
+    <Drawer onToggle={() => setOpen(!open)} isOpen={open}>
+      <Drawer.Head>
+        <DrawerHeadContent>Click me to open</DrawerHeadContent>
+      </Drawer.Head>
+      <Drawer.Body>
+        <DrawerBodyContent>Drawer Content here</DrawerBodyContent>
+      </Drawer.Body>
+    </Drawer>
+  )
+}
+
+const ControlledButtonOnly = () => {
+  const [open, setOpen] = useState(false)
+  return (
+    <>
+      <button onClick={() => setOpen(!open)}>toggle drawer</button>
+      <Drawer isOpen={open}>
+        <Drawer.Head>
+          <DrawerHeadContent>Clicking me won't toggle drawer</DrawerHeadContent>
+        </Drawer.Head>
+        <Drawer.Body>
+          <DrawerBodyContent>Drawer Content here</DrawerBodyContent>
+        </Drawer.Body>
+      </Drawer>
+    </>
+  )
+}
+storiesOf('drawer:rc', module)
+  .addDecorator(story => <div style={{ padding: 48 }}>{story()}</div>)
+  .add('default', () => {
+    return (
+      <Drawer>
+        <Drawer.Head>
+          <DrawerHeadContent>Click me to open</DrawerHeadContent>
+        </Drawer.Head>
+        <Drawer.Body>
+          <DrawerBodyContent>
+            Ambulance and emergency sirens in New York City are wailing
```

Comment:
some heavy filler text...

Reply:
Yeah I was just copying the first few paragraphs text from the last news article I had open didn't even realize it until now.